### PR TITLE
Updated "Delete permanently" dialog

### LINKED ISSUE #133214: Delete Permanently doesn't delete permanently
Does this issue occur when all extensions are disabled?: I haven't tried without extensions, but AFAIK, extensions don't modify the Delete file dialog

- VS Code Version: `1.60.0 (user setup)`
- OS Version: Windows 10, 21h1 (OS Build 19043.1202), using VS Code inside of WSL2 Ubuntu 20.04 LTS

Steps to Reproduce:

1. Create a file (I named it `HelloWorld.java`, but the name and contents shouldn't matter)
2. Navigate to the Explorer tab on the right (the list of files)
3. Right-click on `HelloWorld.java` and click on `Delete Permanently
image
4. Observe the displayed window

Delete permanently is wrong. Even after selecting `Delete` on the dialog above, I can still undo the delete by doing <kbd>CTRL + Z</kbd>.

Expected behavior:
Undoing the delete with <kbd>CTRL + Z</kbd> doesn't undo the delete.

Suggested Fix:
Either make it actually permanently delete, or update the dialog to just say delete

### PR DESCRIPTION
**Pull request by joeljosedev**
This PR fixes #133214

### CODE REVIEW

**Comment by gjsjohnmurray** [comment]
See https://github.com/microsoft/vscode/issues/133214#issuecomment-921483758

File: src/vs/workbench/contrib/files/browser/fileActions.ts [hunk 1]
```diff
@@ -251,7 +251,7 @@ function getMoveToTrashMessage(distinctElements: ExplorerItem[]): { message: str
 	}
 
 	if (distinctElements[0].isDirectory) {
-		return { message: nls.localize('confirmMoveTrashMessageFolder', "Are you sure you want to delete '{0}' and its contents?", distinctElements[0].name), detail: '' };
+		return { message: nls.localize('confirmMoveTrashMessageFolder', "Are you sure you want to delete '{0}' and its contents?", distinctElements[0].name), detail: '(You will not be able to restore it after exiting VS Code)' };
```
Comment (gjsjohnmurray) [inline src/vs/workbench/contrib/files/browser/fileActions.ts:254]: This seems to be a "move to trash" message, in which case it's not true that you won't be able to restore it (the folder) after exiting VS Code

File: src/vs/workbench/contrib/files/browser/fileActions.ts [hunk 2]
```diff
@@ -283,7 +283,7 @@ function getDeleteMessage(distinctElements: ExplorerItem[]): { message: string,
 		return { message: nls.localize('confirmDeleteMessageFolder', "Are you sure you want to permanently delete '{0}' and its contents?", distinctElements[0].name), detail: '' };
 	}
 
-	return { message: nls.localize('confirmDeleteMessageFile', "Are you sure you want to permanently delete '{0}'?", distinctElements[0].name), detail: '' };
+	return { message: nls.localize('confirmDeleteMessageFile', "Are you sure you want to delete '{0}'?", distinctElements[0].name), detail: '' };
```
Comment (gjsjohnmurray) [inline src/vs/workbench/contrib/files/browser/fileActions.ts:286]: Is this a message where you should add the warning about not being able to restore after exit?

File: src/vs/workbench/contrib/files/browser/fileActions.ts (hunk 1 above)
Comment (joeljose350) [reply src/vs/workbench/contrib/files/browser/fileActions.ts:254]: Oh sorry that was a mistake, I didn't notice the `confirmMoveTrashMessageFolder` there.

File: src/vs/workbench/contrib/files/browser/fileActions.ts (hunk 2 above)
Comment (joeljose350) [reply src/vs/workbench/contrib/files/browser/fileActions.ts:286]: Yep, that was the idea.

**Pull request closed without merging**